src/utils/date-helper.ts

```diff
@@ -1,6 +1,10 @@
+import { isDate, isEmpty } from 'lodash'
 // Converts date to string and pacific date string
 // Example Output: August 11, 2023
 export function shortPacificDate (date: Date | string): string {
+  // safety check
+  if (!isDate(date) || isNaN(date.getTime())) return null
```

Comment:
added a safety check

Reply:
Will this fail if date is passed in as an empty string?

Reply:
I see you just verified that in the unit test below 😛